src/backend/distributed/README.md

```diff
@@ -1810,6 +1808,14 @@ GetDistributeObjectOps(Node *node)
 ...
 ```
 
+Finally, when adding support for propagation of a new DDL command, you also need to make sure that:
+* Use `quote_identifier()` or `quote_literal_cstr()` for the fields that might need escaping some characters or bare quotes when deparsing a DDL command.
+* The code is tolerant to nullable fields within given `Stmt *` object, i.e., the ones that Postgres allows not specifying at all.
+* You register the object into `pg_dist_object` if it's a CREATE command and you delete the object from `pg_dist_object` if it's a DROP command.
```

Comment:
I feel like this bullet should mention `markDistributed` since we remove its previous mention now.